charts/zenko/values.yaml

```diff
@@ -95,4 +95,25 @@ redis-ha:
     create: true
   replicas:
     servers: *nodeCount
-    sentinels: *nodeCount
\ No newline at end of file
+    sentinels: *nodeCount
+
+grafana:
+  enabled: false
+  # When 'grafana.enabled' is true, a grafana chart is deployed as part of the Zenko stack
+  ingress:
+    enabled: true
+    hosts:
+      - grafana.local
+  sidecar:
+    datasources:
+      enabled: true
+      # Every config map with the following label will be used as datasource. By default,
+      # promtheus is set as datasource under zenko/templates/datasources.yaml
+      label: grafana-datasource
+    dashboards:
+      enabled: true
+      # Likewise, every config map with the following label will be used as a source for
+      # a dashboard.
+      label: grafana-dasboard
+  adminUser: deployment-specific-admin-user
+  adminPassword: deployment-specific-admin-password
```

Comment:
No. Don't hard-code 'default' passwords. If the Grafana chart doesn't somehow set up some random password in a secret unless overridden, then that should be added upstream. Otherwise, we should simly use that functionality.

Reply:
Yes, it does have support for it.